config/locales/en.yml

```diff
@@ -829,72 +829,72 @@ en:
           "yes": "Bridge"
         building:
           apartment: "Apartment"
-          apartments: "Apartments"
+          apartments: "Apartment Building"
           barn: "Barn"
           bungalow: "Bungalow"
           cabin: "Cabin"
-          chapel: "Chapel"
+          chapel: "Chapel Building"
           church: "Church Building"
           civic: "Civic Building"
           college: "College Building"
           commercial: "Commercial Building"
-          construction: "Building under Construction"
+          construction: "Building Under Construction"
           cowshed: "Cowshed"
           detached: "Detached House"
           dormitory: "Dormitory"
           duplex: "Duplex House"
-          farm: "Farm House"
-          farm_auxiliary: "Auxiliary Farm House"
+          farm: "Farmhouse"
+          farm_auxiliary: "Farm Building"
           garage: "Garage"
           garages: "Garages"
           greenhouse: "Greenhouse"
-          hangar: "Hangar"
+          hangar: "Hangar Building"
           hospital: "Hospital Building"
           hotel: "Hotel Building"
           house: "House"
           houseboat: "Houseboat"
           hut: "Hut"
           industrial: "Industrial Building"
           kindergarten: "Kindergarten Building"
-          manufacture: "Manufacturing Building"
+          manufacture: "Industrial Production Building"
```

Comment:
I recommend `Factory Building`

Reply:
The [`building=manufacture`](https://wiki.openstreetmap.org/wiki/Tag:building%3Dmanufacture) documentation says this is narrower than `building=industrial` but broader than [`building=factory`](https://wiki.openstreetmap.org/wiki/Tag:building%3Dfactory), encompassing storage buildings. I’m not sure why id-tagging-schema felt the need to call it something longwinded; “Manufacturing Building” seems OK given the other overlapping terms.